tox.ini

```diff
@@ -36,4 +36,5 @@ max-line-length = 120
 [gh-actions]
 python =
     3.7: py37
-    3.8: py38, lint, type
\ No newline at end of file
+    3.8: py38, lint, type
```

Comment:
Nit: type checking is expensive. Thus, let's only type check (and) lint for one python version.